recipes/jalview/jalview.sh

```diff
@@ -28,4 +31,11 @@ if [[ "$JALVIEW_JRE" != "j11" && "$JALVIEW_JRE" != "j1.8" ]]; then
   if [[ $( java -version 2>&1 | grep '"1.8' ) != "" ]]; then JALVIEW_JRE=j1.8; fi
 fi
 
-java -jar $DIR/jalview-all-${JALVIEW_JRE}.jar ${@};
+# check if memory maximum is set and if so forward to java-based jalview call
+if [ -z ${JALVIEW_MAXMEM+x} ]; then
```

Comment:
I'd use `-z "${JALVIEW_MAXMEM}"` (no `+x`), unless you really want `JALVIEW_MAXMEM="" jalview.sh` to call `java -Xmx -jar ...`?

Reply:
True enough, check for emptiness should do, was checking whether the var is set at all. Will change.